packages/web-app-files/src/composables/actions/spaces/useSpaceActionsUploadImage.ts

```diff
@@ -56,6 +56,14 @@ export const useSpaceActionsUploadImage = ({
       extraHeaders['X-OC-Mtime'] = '' + file.lastModified / 1000
     }
 
+    if (!store.getters['Files/files'].find((file) => file.name === '.space')) {
```

Comment:
You need to query for the folder via API because it's not guaranteed that files are loaded in the store. That's the case e.g. on the Spaces overview (not inside a specific space).

It seems we already have such logic in other actions, see here for example: https://github.com/owncloud/web/blob/master/packages/web-pkg/src/composables/actions/files/useFileActionsSetImage.ts#L30C1-L30C1

Reply:
Got it. Updated, added update to drive, thus readme is displaying properly in space as well.